omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
@guzzilar No, it is broken. The last part of the URL, `/payment`, is consistent with the file name.

The screenshot below shows the error result page after changed a file name from `payment.php` to `OmisePaymentModuleFrontController.php`.

<img width="991" alt="404_error_-_omise-prestashop" src="https://cloud.githubusercontent.com/assets/4145121/22188131/9d8653b6-e144-11e6-9704-90bb72d23d94.png">

Did you test and is it work on your environment?